src/skmultiflow/trees/nodes/sst_active_learning_node.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+from skmultiflow.trees.nodes import ActiveLearningNodePerceptronMultiTarget
+from skmultiflow.trees.attribute_observer import NumericAttributeRegressionObserverMultiTarget
+from skmultiflow.trees.attribute_observer import NominalAttributeRegressionObserver
+from skmultiflow.utils.utils import get_dimensions
+
+
+class SSTActiveLearningNode(ActiveLearningNodePerceptronMultiTarget):
```

Comment:
Please add a short description.

Reply:
Documentation updated in commit [5b724c9](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5b724c98a7bcc2d270f1034185b2978201235576).